vimeo-networking/src/main/java/com/vimeo/networking/VimeoClient.java

```diff
@@ -922,10 +953,13 @@ public void failure(VimeoError error) {
     // -----------------------------------------------------------------------------------------------------
     // <editor-fold desc="Editing (Video, User)">
     @Nullable
-    public Call<Object> editVideo(String uri, String title, String description, @Nullable String password,
-                                  @Nullable Privacy.PrivacyValue privacyValue,
-                                  @Nullable HashMap<String, Object> parameters,
-                                  ModelCallback<Object> callback) {
+    public Call<Video> editVideo(String uri,
+                                 String title,
+                                 String description,
+                                 @Nullable String password,
+                                 @Nullable Privacy.PrivacyValue privacyValue,
+                                 @Nullable HashMap<String, Object> parameters,
+                                 VimeoCallback<Video> callback) {
```

Comment:
I would also agree that we should use `NotNull` wherever possible. It's much more explicit. We might also look into https://www.jetbrains.com/help/idea/parametersarenonnullbydefault-annotation.html if people don't want to have to specify NotNull manually.